modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/UpgradeLayoutFeatures.kt

```diff
@@ -136,7 +152,25 @@ internal fun UpgradeLayoutFeatures(
             }
 
             if (canUpgrade) {
-                SubscribeButton(onClickSubscribe)
+                Column(
+                    modifier = Modifier,
+                    verticalArrangement = Arrangement.Center,
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                ) {
+                    val bottomPadding = if (shouldShowOffer) 0.dp else 50.dp
+
+                    SubscribeButton(onClickSubscribe, bottomPadding)
+
+                    if (shouldShowOffer) {
+                        offerText?.let {
+                            TextP50(
+                                text = it,
+                                fontWeight = FontWeight.W400,
+                                modifier = Modifier.padding(bottom = 50.dp),
+                            )
```

Comment:
```suggestion
                            TextP50(
                                text = it,
                                fontWeight = FontWeight.W400,
                            )
                            Spacer(Modifier.height(50.dp))
```
It might be nicer to use the Spacer component in this case.

Reply:
[73d7697](https://github.com/Automattic/pocket-casts-android/pull/2674/commits/73d76973fa4f01150797fc9011c6f56227f77a5d)